src/OSPSuite.Core/Domain/DataFactory.cs

```diff
@@ -97,11 +98,22 @@ private DataColumn createColumn(BaseGrid baseGrid, IQuantity quantity, VariableV
       /// </returns>
       private BaseGrid createTimeGrid(double[] values)
       {
-         return new BaseGrid(Constants.TIME, _dimensionFactory.Dimension(Constants.Dimension.TIME))
+         return new BaseGrid(TIME, _dimensionFactory.Dimension(Constants.Dimension.TIME))
          {
-            QuantityInfo = new QuantityInfo(new List<string> {Constants.TIME}, QuantityType.BaseGrid),
+            QuantityInfo = new QuantityInfo(new List<string> {TIME}, QuantityType.BaseGrid),
             Values = new List<float>(values.ToFloatArray())
          };
       }
+
+      //Exposing this one publicly so that we can test it (hard to test with SimModel being involved in the mix)
+      public float CalculateComparisonThreshold(IQuantity quantity, double defaultThreshold)
```

Comment:
@rwmcintosh @Yuri05 
We cannot mock SImModel stuff I think so I am exposting this one publically so that I can test without having to instantiate a simmodel that would create a threshold that is below this value